drltrace_src/drltrace_utils.cpp

```diff
@@ -41,6 +42,22 @@ file_t f_global = INVALID_FILE;
 int reported_disk_error;
 uint op_ignore_asserts = false;
 
+
+/* A faster(?) version of strcmp(), since strcmp() does extra string
+ * comparison we don't need (we just need an equality test).  Returns
+ * 0 when strings are equal, otherwise returns non-zero. */
+inline int
```

Comment:
I bet this compilation problem happens because of ```inline```. You can implement the function in the [header](https://stackoverflow.com/questions/19162732/access-an-inline-function-from-another-file) in this case or remove it. Not sure if it actually improves performance. Up to you...